src/components/shared/renderers/Code.jsx

```diff
@@ -0,0 +1,27 @@
+//@flow
+import React, {useState, useEffect} from 'react';
+import syntaxHighlighting from '../../../helpers/syntax-highlighting';
+
+type Props = {
+  value: string,
+  language: string,
+};
+
+const Code = ({value, language}: Props) => {
+  const [highlight, setHighlight] = useState('');
+
+  useEffect(() => {
+    setHighlight(syntaxHighlighting(value, language));
+  }, [value, language]);
+
+  return (
+    <pre>
+      <code
+        className={`language-${language}`}
+        dangerouslySetInnerHTML={{__html: highlight}}
```

Comment:
I'm actually a little concerned about this... let me try and think of a better way around this before merging. Testing this locally creates a valid html element, but it doesn't execute anything...

```
<script>alert("I'm alive");</script>    
```

<img width="347" alt="Screen Shot 2019-10-25 at 12 40 46 PM" src="https://user-images.githubusercontent.com/9825605/67592177-af184f00-f724-11e9-8ca5-9d31ec4145db.png">